forms/consent/KPCO_Consent_V0.03_Span.html

```diff
@@ -493,8 +493,7 @@ <h1 style="padding-top: 10pt;padding-left: 20pt;text-indent: 0pt;text-align: lef
             <p style="text-indent: 0pt;text-align: left;"><br /></p>
             <h1 style="padding-top: 10pt;padding-left: 20pt;text-indent: 0pt;text-align: justify;">¿QUÉ SUCEDERÁ DURANTE EL ESTUDIO?</h1>
             <p style="padding-top: 6pt;padding-left: 20pt;text-indent: 0pt;text-align: justify;">Si decide participar en Connect, se le pedirá que utilice su número de teléfono o su dirección de correo electrónico para crear una cuenta de MyConnect.  En MyConnect, se le pedirá que:</p>
-            <h1 style="padding-top: 6pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">Brinde información personal. <span class="p">Dicha información incluirá datos tales como su nombre, su fecha de nacimiento, su sexo, su número de seguro social, su domicilio particular y otra información de contacto.</span></h1>
-            <p style="padding-left: 20pt;text-indent: 0pt;text-align: left;">Brindar su número de seguro social es opcional, y puede participar en Connect aun cuando opte por no brindarlo.   Cada cierto tiempo, le preguntaremos si cambió de domicilio, dirección de correo electrónico o número de teléfono, para poder mantenernos en contacto con usted.</p>
+            <h1 style="padding-top: 6pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">Brinde información personal. <span class="p">Dicha información incluirá datos tales como su nombre, su fecha de nacimiento, su sexo, su número de seguro social, su domicilio particular y otra información de contacto. Brindar su número de seguro social es opcional, y puede participar en Connect aun cuando opte por no brindarlo.   Cada cierto tiempo, le preguntaremos si cambió de domicilio, dirección de correo electrónico o número de teléfono, para poder mantenernos en contacto con usted.</span></h1>

```

Comment:
Not sure if the extra spacing was intentional:
```suggestion
            <h1 style="padding-top: 6pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">Brinde información personal. <span class="p">Dicha información incluirá datos tales como su nombre, su fecha de nacimiento, su sexo, su número de seguro social, su domicilio particular y otra información de contacto. Brindar su número de seguro social es opcional, y puede participar en Connect aun cuando opte por no brindarlo. Cada cierto tiempo, le preguntaremos si cambió de domicilio, dirección de correo electrónico o número de teléfono, para poder mantenernos en contacto con usted.</span></h1>

```